docker/Dockerfile

```diff
@@ -6,13 +6,32 @@ RUN apk add --update git && \
     apk add curl && \
     apk add bash && \
     apk add openjdk8 && \
-    apk add openjdk8-doc
+    apk add openjdk8-doc && \
+    apk add expect
 
 # Set sbt and java env vars
 ENV SBT_VERSION 0.13.11
 ENV SBT_HOME /usr/local/sbt
 ENV PATH ${PATH}:${SBT_HOME}/bin
 ENV PATH ${PATH}:/usr/lib/jvm/java-1.8-openjdk/bin
+ENV SCALA_VERSION=2.11.8
+ENV SCALA_HOME=/usr/share/scala
```

Comment:
In the PR https://github.com/lampepfl/dotty/pull/1951 I've a script that depends on `scala`, and I'm thinking the scala command can also be generally useful for diagnostics if we want to investigate problems with the docker -- e.g. to compare behavior with `bin/dotc`.